test/jest/playwrightEnvironment.js

```diff
@@ -53,11 +55,26 @@ class PlaywrightEnvironment extends NodeEnvironment {
 
   async setup() {
     await super.setup();
+    const {coverage, uninstall} = installCoverageHooks(browserName);
+    this.coverage = coverage;
+    this.uninstallCoverage = uninstall;
   }
 
   async teardown() {
     await this.fixturePool.teardownScope('worker');
     await super.teardown();
+    // The setup might have failed for some reason.
```

Comment:
How come?

Reply:
We don't expect the setup to throw an error, but if it does, jest immediately calls teardown. Teardown then throws because `this.coverage` is `undefined`. Jest only reports the second error, and whoever is debugging has a bad time. Updated the comment.